tools/CMakeLists.txt

```diff
@@ -39,3 +39,7 @@ if ( CMAKE_SYSTEM MATCHES Linux )
     message ("-- Building start-stop-daemon")
     HPCC_ADD_SUBDIRECTORY (start-stop-daemon)
 endif()
+
```

Comment:
Would be preferable to say

```
  if ( CMAKE_SYSTEM MATCHES "Linux|APPLE" )
```
above (and at the same time update the comment that says start-stop-daemon is linux only)